.github/actions/build-test/windows/action.yml

```diff
@@ -0,0 +1,168 @@
+name: Build & Test - Windows
+
+inputs:
+  GH_TOKEN:
+    required: true
+  BOOST_VERSION:
+    required: true
+  THRIFT_VERSION:
+    required: true
+  BUILD_TYPE:
+    required: true
+  SHARED_LIBS_TOGGLE:
+    required: true
+  OPENSSL_TOGGLE:
+    required: true
+  ARCH_CHOCO_OPTIONS:
+    required: true
+  ARCH_ADDRESS_MODEL:
+    required: true
+  ARCH_CMAKE:
+    required: true
+  INSTALL_BOOST:
+    required: true
+  INSTALL_THRIFT:
+    required: true
+  BOOST_URL:
+    required: true
+  BOOST_ARCHIVE_NAME:
+    required: true
+  BOOST_FOLDER_NAME:
+    required: true
+  BOOST_INCLUDE_FOLDER:
+    required: true
+  RUN_TESTS:
+    required: true
+  HAZELCAST_ENTERPRISE_KEY:
+    required: true
+  AWS_ACCESS_KEY_ID:
+    required: true
+  AWS_SECRET_ACCESS_KEY:
+    required: true
+  HZ_TEST_AWS_INSTANCE_PRIVATE_IP:
+    required: true
+
+env:
+  # Not possible to set this as a default
+  # https://github.com/orgs/community/discussions/46670
+  shell: pwsh
+
+runs:
+  using: composite
+  steps:
+    - name: Read Config
+      shell: bash
+      run: cat .github/config.env >> $GITHUB_ENV
+
+    - name: Download hazelcast-enterprise-tests.jar
+      shell: ${{ env.shell }}
+      run: |
+        [Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12
+        Invoke-WebRequest https://raw.githubusercontent.com/hazelcast/private-test-artifacts/data/certs.jar -Headers @{"Authorization"="token ${{ inputs.GH_TOKEN }}"} -OutFile hazelcast-enterprise-${{ env.HAZELCAST_VERSION }}-tests.jar
```

Comment:
too long. Split to multiline

Reply:
0e1ecfacd43fc20338097e87bbbb3026cf3afdc8